test/euclidean.jl

```diff
@@ -229,24 +229,27 @@ using Manifolds: induced_basis
     @testset "Euclidean metric tests" begin
         M = Euclidean(2)
         p = zeros(2)
-        C1 = christoffel_symbols_first(M, p)
+        A = Manifolds.get_default_atlas(M)
```

Comment:
This is a trivial atlas for sure, but just these lines 232-234 look _so_ nice in code now. I just wrote a similar exercise recently and I might (just might, not yet sure) try to write an atlas tutorial on a subset of the Euclidean manifold based on that.

Reply:
Thanks, this finally lets us take metrics seriously :slightly_smiling_face: . For a bit more complicated atlas you may look at `StereographicAtlas` on sphere (this may make a bit nicer tutorial).

Reply:
Sure that might be nicer since it is not just one chart and one can explain a bit more.